templates/geosafe/analysis/options_panel.html

```diff
@@ -117,7 +117,8 @@ <h4 class="list-title">Select an impact layer</h4>
         <div id="summary-option" class="option">
             <a href="javascript:show_analysis_summary()"
                 title="Show Analysis Summary">
-                <i class="fa fa-file"></i>
+                <!--i class="fa fa-file"></i -->
+                <img src="{% static "geosafe/img/show-summary2.svg" %}" alt="Show Analysis Summary" class="svg"/>
```

Comment:
please rename `show-summary2.svg` to just `show-summary.svg` the number is not relevant.